src/ar_key_db.erl

```diff
@@ -1,13 +1,12 @@
 -module(ar_key_db).
--export([start/0, get/1, put/2, maybe_add/1, remove/1]).
+-export([start/0, get/1, put/2]).
 -compile({no_auto_import, [{get, 1}, {put, 2}]}).
 -include_lib("eunit/include/eunit.hrl").
 -include("ar.hrl").
-%%% Defines a small in-memory metadata table for Arweave nodes.
-%%% Typically used to store small peices of globally useful information
-%%% (for example: the port number used by the node).
+%%% Database for storing decryption keys from encrypted recall blocks. The
+%%% entries has a TTL. The DB is a singleton.
 
-%% @doc Initialise the metadata storage service.
+%% @doc Create the DB. This will fail if a DB already exists.
```

Comment:
Not a big deal, but it should be `a DB` the first time and `the DB` the second, same below.

Reply:
✅